wrappers/wrapper.rb

```diff
@@ -23,9 +23,7 @@ def initialize(hash = {})
     end
 
     def solver_constraints
-      [
-       :assert_no_pickup_timewindows_after_delivery_timewindows,
```

Comment:
I think this should still be checked.

Reply:
As the consistency of timewindows within relations is checked within validate_data in a more generic way : `check_relation_consistent_timewindows` we are sure that this situation cannot happen here